sdks/java/io/elasticsearch/src/main/java/org/apache/beam/sdk/io/elasticsearch/ElasticsearchIO.java

```diff
@@ -202,7 +202,7 @@ static void checkForErrors(HttpEntity responseEntity, int backendVersion, boolea
         } else {
           if (backendVersion == 2) {
             errorRootName = "create";
-          } else if (backendVersion == 5 || backendVersion == 6) {
+          } else if (backendVersion == 5 || backendVersion == 6 || backendVersion == 7) {
```

Comment:
Why not just use simple "if/else" condition in this case? Like:
```
errorRootName = (backendVersion == 2) "create" ? "index";
```

Reply:
better ton avoid running on non-tested versions (if we use >)